rabbitmq/datadog_checks/rabbitmq/openmetrics/check.py

```diff
@@ -1,5 +1,13 @@
 from datadog_checks.base import OpenMetricsBaseCheckV2
 
+from .metrics import METRICS_RENAMES
+
 
 class RabbitMQOpenMetrics(OpenMetricsBaseCheckV2):
     __NAMESPACE__ = "rabbitmq"
+
+    def configure_scrapers(self):
```

Comment:
Seems that the logic for checking if `include_aggregated_endpoint` is enabled is missing.
I assume `include_aggregated_endpoint` is the default metric collection option?

Reply:
Oh, sorry I missed this somehow. Actually, I left it out just so as to focus on one piece of functionality at a time.

It appears in a subsequent PR.